packages/framework/src/Component/Image/Processing/ImageThumbnailFactory.php

```diff
@@ -20,7 +22,7 @@ public function __construct(protected readonly ImageProcessor $imageProcessor)
      * @param string $filepath
      * @return \Intervention\Image\Image
      */
-    public function getImageThumbnail($filepath)
```

Comment:
isn't possible to use imgproxy even in the administration and avoid creating a thumbnail?

Reply:
yeah, I looked into it but it felt out of the scope of this PR, the thumbnails (and domain icons) are not processed via imgProxy as they are located in `content/admin/images/`. It would require the nginx configuration change and I was not sure whether it made sense now as we plan to change the administration completely :thinking: Let's discuss it further, please

Reply:
discussed and yes, it is possible! :slightly_smiling_face: